shared/Microsoft.AspNetCore.StaticFiles.ByteRangeHelper.Sources/ByteRangeHelper.cs

```diff
@@ -5,9 +5,9 @@
 using System.Collections.Generic;
 using Microsoft.Net.Http.Headers;
 
-namespace Microsoft.AspNetCore.StaticFiles.Infrastructure
+namespace Microsoft.AspNetCore.StaticFiles
```

Comment:
You could change the ns to `Microsoft.Net.Http.Headers`. It doesn't have much to do with StaticFiles apart from it being located here.